ping_federate/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - ping_federate
+
+## 1.0.0 / 2024-06-17
+
+***Added***:
+
+* Initial Release
```

Comment:
```suggestion
<!-- towncrier release notes start -->
```

@akanshi-crest sorry about all the confusion with this. In the end it turns out we need to treat this like a Python package, which means the changelog will be generated automatically.

For your next integration, please make sure to upgrade `ddev` to the latest version, that should have all the scaffolding working properly.